lib/pbench/server/cache_manager.py

```diff
@@ -119,7 +119,7 @@ def __post_init__(self, dir_path, path):
         self.name = path.name
         self.location = path.relative_to(dir_path)
         if path.is_symlink():
-            self.type = CacheType.SYMLINK.name
+            self.type = CacheType.SYMLINK
```

Comment:
Cool. I'm not sure whether you'll get a complaint, but technically your `type` should now be `type: CacheType` or maybe `type: Optional[CacheType] = None`?

Actually, what's there now is `type: str = CacheType` which is ... odd, since your default "string" value is the `CacheType` class.

Reply:
Nobody says that type hints get checked or enforced by anything....

Reply:
Well, technically, all of the `= None` type hints ought to be `Optional[]`, but I won't nit-pick this further, and I'm resolving this. 😆

Reply:
Yep, I see the warnings in PyCharm, too...and I would be happier if the code were "clean" (although, it objects to some things that we aren't going to fix, so there's that challenge).

> all of the `= None` type hints ought to be `Optional[]`

I mentioned that elsewhere; I'm hoping that this will get addressed in the course of another change.  😉